backend/processing/check_files_checksums/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
```

Comment:
We should use Ubuntu 20.04 - it's the latest LTS, so it'll be supported for longer.

Reply:
Fixed in 3476dd6a758df5ad18cb67bf4fce299420dc0f6c